pkg/apiserver/api/v1/instance.proto

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+import "meta.proto";
+
+message Instance {
+  ObjectMeta metadata = 1;
+  string spec = 2;
+  string status = 3;
+}
+
+message InstanceList {
+  ListMeta metadata = 1;
+  repeated Instance items = 2;
+}
+
+message InstanceGetRequest {
+  string instance = 1;
+  string version = 2;
+  string name = 3;
+  string account = 4;
+}
+
+message InstanceDeleteRequest {
+  string instance = 1;
```

Comment:
What's the difference between `instance` and `name`?

Reply:
same here
so, `instance` it's service/offering name, i.e. `couchdbs.eu-west-1.team-a`, `name` that we get from `offering.spec.crd.name`. we can rename this field to `offering` or back to `service`.
and `name` is the name of the instance that created from this offering, like `ProdCouchDB`